onnx_tf/common/handler_helper.py

```diff
@@ -0,0 +1,64 @@
+import re
+import warnings
+
+from onnx import defs
+
+from . import op_name_to_lower
+from onnx_tf.handlers.frontend import *  # noqa
+from onnx_tf.handlers.frontend_handler import FrontendHandler
+
+
+def __get_all_subclasses(clazz, except_regex=None):
+  all_subclasses = set(clazz.__subclasses__()).union([
+      s for c in clazz.__subclasses__()
+      for s in __get_all_subclasses(c, except_regex=except_regex)
+  ])
+  if except_regex is not None:
+    all_subclasses = set(
+        filter(lambda c: re.match(except_regex, c.__name__) is None,
+               all_subclasses))
+  return all_subclasses
+
+
+def get_all_frontend_handlers(opset_dict):
+  handlers = {}
+  for handler in __get_all_subclasses(
+      FrontendHandler, except_regex=r'.*Common$'):
```

Comment:
Can we have some documentations around the definition of FrontendHandler saying that any subclass of FrontendHandler with suffix Common is not treated as an op handler?

TBH, I think your previous way of using register_op decorator was more expressive and friendly in expressing what we want to achieve. Anyways, it's up to you to decide; there's kind of a trade of between the extent of automation and semantic expressiveness.